src/server/model_db.ts

```diff
@@ -20,7 +20,11 @@ import * as fs from 'fs';
 import * as path from 'path';
 import {promisify} from 'util';
 
+import {setupUserModel} from '../model';
+import {jsonToTensor, TensorJson, tensorToJson} from '../serialization';
+
 const DEFAULT_MIN_UPDATES = 10;
+const NO_MODELS_IN_FOLDER = '0';
```

Comment:
This is a little bit hacky -- I'm having `getLatestId` return `'0'` if there are no models in the data directory, and then using that as a special indicator that we need to initialize the database. Let me know if you think of a nicer way to structure this. I could have `getLatestId` return `undefined` instead, I guess.